constellations/orion/src/repos/migrations/000030_add_school_at_users.up.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users
+    ADD COLUMN school varchar(32);
```

Comment:
Hmm turns out "Winston churchill High school" is 29 characters long which is a bit too close for comfort.
Let's make this 64.